src/PlaneWaveBasis.jl

```diff
@@ -89,10 +89,15 @@ struct PlaneWaveBasis{T,
     ## These fields are not actually used in computation, but can be used to reconstruct a basis
     # Monkhorst-Pack grid used to generate the k-points, or nothing for custom k-points
     kgrid::AbstractKgrid
-    # full list of (non spin doubled) k-point coordinates in the irreducible BZ
+    # full list of (non spin doubled) k-point coordinates in the irreducible BZ (duplicates possible)
```

Comment:
Can you also add a comment that these are better accessed through the functions you introduce below
to take care of the duplication.